api-spec/protobuf/ark/v1/service.proto

```diff
@@ -54,6 +54,12 @@ service ArkService {
       get: "/v1/info"
     };
   }
+  rpc ReverseBoardingAddress(ReverseBoardingAddressRequest) returns (ReverseBoardingAddressResponse) {
+    option (google.api.http) = {
+      post: "/v1/reverseboarding"
+      body: "*"
+    };
+  };
```

Comment:
It is for sure handy to have server to generate the address, but we also need to give possibility to build the script "offline" assuming user after first connection caches the descriptor template accepted (ideally we could have an expiry time also attached to each descriptor template). Not needed for now, but I think we should include a way to enumerate also the script types of boarding address. @altafan

Reply:
in the pending spec change in #228 we were indeed proposing to pass the descriptor and make the Onboard to be a RPC to be called AFTER the tx is sent to the static address. 



```proto
message OnboardRequest {
  string boarding_txid = 1;
  string descriptor = 2;
}
```

maybe we could still remove the Onboard RPC, and simply add a dedicated `BoardingUtxo` msg as accepted input for a payment, which will include both txid, descriptor. Server can then easily accept/reject registraton based on the descriptor

we need to add a `supported_boarding_script_templates` to `GetInfo`  so clients can catch that at first connection, and cache it

Reply:
actually, for the covenant and covenat-less versions, the script for this kind of onboarding is the same (`A + S or A after 4 weeks`) so we don't really need the enumeration for now.

We don't really need any new rpc at all if we move the script/address creation client-side.